src/tests/class_tests/openms/source/MSSpectrum_test.cpp

```diff
@@ -811,7 +811,84 @@ START_SECTION((void sortByPosition()))
       TEST_EQUAL(true,false)
     }
   }
+}
+END_SECTION
+
+
+START_SECTION((void sortByPositionPresorted()))
+{
+  MSSpectrum ds;
+  Peak1D p;
+  MSSpectrum::FloatDataArray float_array;
+  MSSpectrum::StringDataArray string_array;
+  MSSpectrum::IntegerDataArray int_array;
+  std::vector<double> mzs, intensities;
+  MSSpectrum::Chunks chunks(ds);
+  intensities.push_back(19);  mzs.push_back(419.113); float_array.push_back(19);  string_array.push_back("19");  int_array.push_back(19);
```

Comment:
you could use initializer lists i.e. `std::vector<double> mzs = { 1, 2, 3};` to init the arrays. Less code and more readable?

Reply:
Tried to do so, just to find out that you can't apply list initialization on FloatDataArrays, as they have that MetaData member...

Reply:
Did it with .assign, now.